registry/polkadot.json

```
@@ -436,5 +436,13 @@
     "shortDescription": "Fluffy Labs is a Wrocław-based organization within the Polkadot ecosystem, focusing on blockchain technology and its evolution.",
     "longDescription": "Fluffy Labs is a Wrocław-based organization within the Polkadot ecosystem, focusing on blockchain technology and its evolution.",
     "isOrganization": true
+  },
+{
+    "address": "12b35i6gjXhMzYU7Dy97RngXbuJngDJ8oeY15aM533mmqGaf",
+    "name": "Dotcast DAO",
+    "image": "https://raw.githubusercontent.com/D0tcast/opengov-delegate-registry/refs/heads/master/images/Dotcast.png",
```

please, use our repo as a source of image
```suggestion
    "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/Dotcast.png",
```